python/dlisio/plumbing/path.py

```diff
@@ -0,0 +1,81 @@
+
+from .basicobject import BasicObject
+from .valuetypes import scalar, vector
+from .linkage import obname
+
+
+class Path(BasicObject):
+    """
+    Path Objects defines Channels in the Data Frames of a given Frame Type and
+    are combined to define part or all of a Data Path, and variation in the
+    alignment.
+
+
+    Attributes
+    ----------
+
+    frame_type : Frame
+        Frame type is defined as a frame object in the current logical file which
+        indicates the Frame Type in which the Channels of the current Path are
+        recorded.
+
+    well_reference_point : Wellref
+        Well reference point reference the Object in the current Logical File
+        that specifies the Well Reference Point for entities specified by the
+         remaining Attributes of the Path Object.
+
+    value : list of Channel
+        Value references a Channel Object that defines the Value Channel for
+        the current Path.
+
+    borehole_depth
+        Specifies the constant Borehole Depth coordinate for the
```

Comment:
Constant borehole depth, may or may not be described by a channel object.
&&
Constant vertical depth, may or may not be described by a channel object.